test/cli/test-fixtures/image-java-spdx-tools/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM cgr.dev/chainguard/jdk
+
+# Not committed to repository - can be found here: https://github.com/spdx/tools-java/releases
+ADD tools-java-1.1.3-jar-with-dependencies.jar /usr/local/bin/tools-java.jar
```

Comment:
Could we just curl/wget this in the Dockerfile from a specific release artifact? (Probably not latest, but also maybe latest)

Reply:
+1 on making the pull in the dockerfile and not needed any files in the docker context

Reply:
updated to pull in the docker file and added cli tests for tag value validation